lib/beaker/dsl/structure.rb

```diff
@@ -54,6 +55,51 @@ def step step_name, &block
         end
       end
 
+      # Provides a method to help manual tests. So we can use beaker to set up
+      # the environment, then prompt a user to manually check the setup.
+      # @param [String] step_name The name of the step to be logged.
+      # @param [String] fail_message The message to return if this manual step failed
+      def manual_step step_name, fail_message='Manual step failed.'
+        logger.notify "\n* #{step_name}\n"
+        if(!@options.has_key?(:exec_manual_tests))
+          # if the option -exec-manual-tests is not set then this has executed outside of a manual tests
+          # so we raise an error to avoid issues
+          raise('--exec-manual-tests option not set, this means a manual_step was used outside a manual_test')
+        end
+
+        set_current_step_name(step_name)
+        # Here we prompt the user to tell us if the step passed or failed
+        loop do 
+          input = Readline.readline('did this step pass, Y/n?', true).squeeze(" ").strip
+          if input == 'Y'
+            break
+          elsif input == 'n'
+            raise Beaker::DSL::FailTest, fail_message
```

Comment:
I think the environment/context for running these will be executing beaker on a laptop/workstation; using rake tasks that are as close as possible to the rake tasks that run the executed tests in CI (use the same vmpooler hosts, use the same package installation, run some or all of the automated tests as well as these special manual tests).

We haven't considered results storage - the workflow for manual testing today is that we generally do not store manual test results; we file tickets for anything that's not right; and everything else has implicitly passed. Maybe in future we would want a results store.